### DISCUSSION TITLE
square button on android phone

### DISCUSSION THREAD

**Discussion by reymalahay** [mobile-dev-inc/Maestro#1552] [Q&A] [answered]
hi all,

how does one make an automation script `press` the square button, i.e., the button used to show,resume or end recent applications.

please advise.

thanks in advance,
rey malahay

**Comment by portyu9** [comment, marked as answer]
Hi @reymalahay 

Maestro currently does not have a `pressKey` value for Android's **Recents / Overview / square** button.

If your real goal is simply to return to your app after putting it in the background, you often don't need Recents. You can do:

```yaml
- pressKey: Home

# ...

- launchApp:
    stopApp: false
```

`stopApp: false` brings the existing backgrounded app back to the foreground without restarting it.

If you specifically need to test the Android **Recents UI itself**, that action currently needs to happen outside the Maestro flow, for example from an ADB wrapper:

```sh
adb shell input keyevent KEYCODE_APP_SWITCH
```

Then continue the Maestro test afterward.

That distinction is important: use `launchApp.stopApp: false` when the objective is application lifecycle/navigation; use external ADB only when testing the actual Android Recents screen is part of the scenario.

Please mark this answer as accepted if it helped, thank you.

**Reply by reymalahay** [reply]
thank you for the response, @portyu9 

i will close this issue.

kind regards,
rey malahay
